docs/basics/101-126-config2.rst

```diff
@@ -0,0 +1,285 @@
+.. _config2:
+
+Everything you ever wanted to configure [2]
+-------------------------------------------
+
+As the last section already suggest, within a Git repository,
+``.git/config`` is not the only configuration file.
+There are also ``.gitmodules`` and ``.gitattributes``, and in DataLad datasets
+there also is a ``.datalad/config`` file.
+
+All of these files store configurations, but have an important difference:
+They are version controlled, and upon sharing a dataset these configurations
+will be shared as well. An example for a shared configuration
+is the one that the ``text2git`` configuration template applied:
+In the shared copy of your dataset, text files are also saved with Git,
+and not Git-annex (see section :ref:`sibling`). The configuration responsible
+for this behavior is in a ``.gitattributes`` file, and we'll start this
+section by looking into it.
+
+``.gitattributes``
+^^^^^^^^^^^^^^^^^^
+
+This file lies right in the root of your superdataset:
+
+.. runrecord:: _examples/DL-101-125-104
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ cat .gitattributes
+
+This looks neither spectacular nor pretty. Also, it does not follow the ``section-option-value``
+organization of the ``.git/config`` file anymore. Instead, there are three lines,
+and all of these seem to have something to do with the configuration of Git-annex.
+There even is one key word that you recognize: MD5E.
+If you have read the hidden section in :ref:`symlink`
```

Comment:
We need to come up with a proper name for that. It is only "hidden" (or toggle-off) in the browser-based variant of the book. Otherwise it would be a box or something like that.

(fine for now, but needs to be kept in mind)